sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Expression.scala

```diff
@@ -95,7 +95,7 @@ abstract class Expression extends TreeNode[Expression] {
     ctx.subExprEliminationExprs.get(this).map { subExprState =>
       // This expression is repeated meaning the code to evaluated has already been added
       // as a function and called in advance. Just use it.
-      val code = s"/* $this */"
+      val code = s"/* ${this.toString.replace("*/", "\\*\\/")} */"
```

Comment:
Should we put this as a utility in ctx?

Maybe. def escapeComment(< thing to put inside /* */>)

This seems easy to get wrong in the future too.